airflow/ui/src/pages/Dashboard/Health/Health.tsx

```diff
@@ -54,15 +54,14 @@ export const Health = () => {
           status={data?.triggerer.status}
           title="Triggerer"
         />
-        {/* TODO: Update this to match the API when we move the config check to the API level */}
-        {data?.dag_processor.status === undefined ? undefined : (
+        {data && "dag_processor" in data ? (
```

Comment:
I think this could be simplified, cf following suggestions. (remove ternary, and optional chaining operator)
```suggestion
        {data?.dag_processor && (
```

Reply:
Done

Reply:
It the code pushed ? I'm not sure I see the change